app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -334,6 +344,27 @@ def parse_persistent_volume(persistent_volume)
       new_result
     end
 
+    def parse_persistent_volume_claim(claim)
+      new_result = parse_base_item(claim)
+      new_result.merge!(
+        :desired_access_modes => claim.spec.accessModes.try(:join, ','),
```

Comment:
@blomquisg @Fryguy @alongoldboim can we use the postgres/rails4 arrays?

http://blog.plataformatec.com.br/2014/07/rails-4-and-postgresql-arrays/